### PR TITLE
Update description and move to stable

### PR DESCRIPTION
**Pull request by croomes**
Add more product info and move from alpha to stable channel.

### CODE REVIEW

File: deploy/stos.subscription.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ spec:
   source: olm-operators
   sourceNamespace: olm
   name: storageos
-  channel: alpha
+  channel: stable
```
Comment (darkowlzz) [inline deploy/stos.subscription.yaml]: This is used in the e2e test setup. And changing this without changing https://github.com/storageos/cluster-operator/blob/master/deploy/olm-operators.configmap.yaml#L471 will fail the operator installation. But I'm working on changing the configmap a bit, so let's remove this change and just change the channel in https://github.com/storageos/cluster-operator/blob/master/deploy/olm/storageos/storageos.package.yaml#L4 only.

File: deploy/olm-operators.configmap.yaml [hunk 2]
```diff
@@ -292,11 +292,11 @@ data:
         minKubeVersion: "1.10.0"
         maturity: alpha
```
Comment (darkowlzz) [inline deploy/olm-operators.configmap.yaml]: Not to worry about changing this. I'm changing this file in another PR I'm working on.

File: deploy/olm/storageos/storageosoperator.0.0.10.clusterserviceversion.yaml [hunk 3]
```diff
@@ -9,24 +9,93 @@ metadata:
...
   # replaces: 
   displayName: Storageos Cluster Operator
   description: |
-    StorageOS transforms commodity server or cloud based disk capacity into enterprise-class storage to run persistent workloads such as databases in containers. The StorageOS Operator installs and manages StorageOS clusters.
+    StorageOS is a software-defined storage platform that transforms commodity
+    server or cloud based disk capacity into enterprise-class storage for
+    running persistent workloads such as databases in containers.
+
+    The StorageOS Cluster Operator installs and manages StorageOS within an
+    Openshift cluster. Cluster nodes may contribute local or attached disk-based
```
Comment (actionbuddha) [inline deploy/olm/storageos/storageosoperator.0.0.10.clusterserviceversion.yaml]: Is this Openshift only or all K8S derivatives?

Comment (croomes) [reply deploy/olm/storageos/storageosoperator.0.0.10.clusterserviceversion.yaml]: Probably more targeted at Openshift at this point.

Comment (darkowlzz) [reply deploy/olm/storageos/storageosoperator.0.0.10.clusterserviceversion.yaml]: All the development and tests, including e2e tests, were done on k8s.

**Pull request merged**